intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/build/IdeaOutputConfigurationProvider.xtend

```diff
@@ -68,5 +74,17 @@ class IdeaOutputConfigurationProvider implements IContextualOutputConfigurationP
 		}
 		return defaultOutput.outputConfigurations
 	}
+	
+	def String toModuleRelativePath(String path, Module module) {
+		val url = new URL(VfsUtil.pathToUrl(path))
+		if (url.toURI.isAbsolute) {
+			ApplicationManager.application.<String>runReadAction[
```

Comment:
The return value of the read action is unused thus the method always returns 'path'?